policy/modules/contrib/qgs.te

```diff
@@ -0,0 +1,74 @@
+policy_module(qgs, 1.0.0)
+
+########################################
+#
+# Declarations
+#
+
+type qgs_t;
+type qgs_exec_t;
+init_daemon_domain(qgs_t, qgs_exec_t)
+
+permissive qgs_t;
+
+type qgs_var_lib_t;
+files_type(qgs_var_lib_t)
+
+type qgs_var_run_t;
+files_type(qgs_var_run_t)
+files_pid_file(qgs_var_run_t)
+files_create_var_run_dirs(qgs_t)
+
+# Common config file for potentially many SGX
+# related apps doing SGX quoting
+type sgx_qcnl_etc_t;
+files_type(sgx_qcnl_etc_t)
```

Comment:
Do you expect these config file types will not (or should not) be read by any other domain? It would be convenient to mark them using files_config_file(), but that gives access to the type to some 20 other domains.

Reply:
I don't believe any of the data in the config file is security sensitive, so it is probably fine to use `files_config_file`.

Reply:
I've removed the 'sgx_qcl_etc_t' type entirely and just leave this file labelled 'etc_t'.